packages/@sanity/core/src/commands/dataset/alias/unlinkAliasHandler.js

```diff
@@ -0,0 +1,59 @@
+import promptForDatasetAliasName from '../../../actions/dataset/alias/datasetAliasNamePrompt'
+import validateDatasetAliasName from '../../../actions/dataset/alias/validateDatasetAliasName'
+import * as aliasClient from './datasetAliasesClient'
+import {ALIAS_PREFIX} from './datasetAliasesClient'
+
+export default async (args, context) => {
+  const {apiClient, output, prompt} = context
+  const [, alias] = args.argsWithoutOptions
+  const client = apiClient()
+
+  const nameError = alias && validateDatasetAliasName(alias)
+  if (nameError) {
+    throw new Error(nameError)
+  }
+
+  const [fetchedAliases] = await Promise.all([aliasClient.listAliases(client)])
```

Comment:
This is only doing a single fetch, so it reads a little better if it's simplified to not use `Promise.all`:
```suggestion
  const fetchedAliases = await aliasClient.listAliases(client)
```